### PR TITLE
[feature]: Adds input to select the number of results showing in the results pane

### PR DESCRIPTION
**Pull request by marmoure**
this is concept for providing an option to select the size of the results showing on the results pane
image
@joewiz  i would love your feedback on this

### CODE REVIEW

**Comment by line-o** [comment]
Looks like a good feature. Will you be able to add a test for that?

**Comment by marmoure** [comment]
> Looks like a good feature. Will you be able to add a test for that?

@line-o i ll see what i can do

File: src/eXide.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ eXide.app = (function(util) {
 	var startOffset = 0;
 	var currentOffset = 0;
 	var endOffset = 0;
+    var numberOfResults = 10;
```
Comment (dizzzz) [inline src/eXide.js:76]: indenting?

**Review by dizzzz — APPROVED** [review]

**Comment by joewiz** [comment]
@marmoure This works great, thank you!

**Pull request merged**